sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -272,12 +272,13 @@ def test_make_recordset(client):
 
 @pytest.mark.remote_data
 def test_search_metadata(client):
-    metadata = client.search_metadata(a.Time('2014-01-01T00:00:00', '2014-01-01T00:02:00'),
-                                      a.jsoc.Series('aia.lev1_euv_12s'), a.jsoc.Wavelength(304*u.AA))
-    assert isinstance(metadata, pd.DataFrame)
-    assert metadata.shape == (11, 176)
-    for i in metadata.index.values:
-        assert (i.startswith('aia.lev1_euv_12s') and i.endswith('[304]'))
+    with pytest.raises(SunpyDeprecationWarning):
```

Comment:
is the warning an exception or a warning? and if so... which bit of the code below does it? only `search_metadata`? Does the code still keep running the rest of the test?

Reply:
it is a depcration warning due to just `search_metadata`. Yes, this test should be removed after next release. Yes, code still supports `search_metadata` with a warning.